.github/workflows/ci.yml

```diff
@@ -102,6 +102,31 @@ jobs:
       - name: Publish coverage report
         uses: codecov/codecov-action@v3
 
+  docs:
+    runs-on: ubuntu-latest
+    name: Documentation build
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          cache: 'pip'
+          cache-dependency-path: 'requirements/*.txt'
+      - name: Install dependencies
+        run: pip install -r requirements/ci.txt pytest
+      - name: Generate environment variable documentation using OAf and check if it was updated
+        run: |
+          python src/manage.py generate_envvar_docs --file docs/installation/config.rst --exclude-group Celery
```

Comment:
Let's have a bash file with it, it can be useful for generating docs
Too bad it seems impossible to add it as sphinx event